F
======================================================================
FAIL: test_small_prime (__main__.SieveTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/furkan/csc209-w19/grading/mark_a3/marking_scripts/test_helpers.py", line 190, in _t
    self.assertEqual(stdout, expected_stdout)
AssertionError: '' != '39 is prime\nNumber of filters = 1'
+ 39 is prime
+ Number of filters = 1

----------------------------------------------------------------------
Ran 1 test in 0.275s

FAILED (failures=1)
